pkg/qos/queue_sort.go

```diff
@@ -25,22 +25,22 @@ import (
 )
 
 // Name is the name of the plugin used in the plugin registry and configurations.
-const Name = "QoSSort"
+const Name = "QOSSort"
 
-// QoSSort is a plugin that implements QoS class based sorting.
-type Sort struct{}
+// QOSSort is a plugin that implements QoS class based sorting.
+type QOSSort struct{}
```

Comment:
It's sometimes conventional to not include the package name (qos) in the type name. Ref: https://github.com/golang/go/wiki/CodeReviewComments#package-names.

I would suggest to keep it as, or later renamed to `GenericSort`.

Reply:
> It's sometimes conventional to not include the package name (qos) in the type name. Ref: https://github.com/golang/go/wiki/CodeReviewComments#package-names.
> 
> I would suggest to keep it as, or later renamed to `GenericSort`.
Agree, changed it back to Sort.

Reply:
If QueueSort is just one of plugins in package "qos" and other QoS related plugins can be added to the package in the future, the package name "qos" makes sense.  I'll submit a PR on the GenericSort plugin based on our discussions.  Thanks.

Reply:
> > The same failed integration test as coscheduling. Did I miss to change something? Testing in our local clusters worked just fine.
> 
> I created issue #19 to track this.

After reverting the QueueSort plugin struct name to "Sort", it passed the integration test.